src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Helpers/SourceWriter.cs

```diff
@@ -11,33 +11,30 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
     internal sealed class SourceWriter
     {
         private readonly StringBuilder _sb = new();
-        private int _indentationLevel;
-
-        public int Length => _sb.Length;
-        public int IndentationLevel => _indentationLevel;
-
         private static readonly char[] s_newLine = Environment.NewLine.ToCharArray();
 
+        public int IndentationLevel;
```

Comment:
Nit: perhaps convert to a property so that it cannot be accidentally set to negative values?